test/extensions/stats_sinks/open_telemetry/open_telemetry_integration_test.cc

```diff
@@ -165,6 +176,7 @@ class OpenTelemetryGrpcIntegrationTest : public Grpc::GrpcClientIntegrationParam
 
   FakeHttpConnectionPtr fake_metrics_service_connection_;
   FakeStreamPtr otlp_collector_request_;
+  std::string stat_prefix_ = "";
```

Comment:
Nit: no need to explicitly initialize the default.

Reply:
Done